apps/pie-docs/src/patterns/cookie-banner/code/props.json

```diff
@@ -14,10 +14,16 @@
             }
         ],
         [
-            "locale",
-            "Any locale from [this list](https://github.com/justeattakeaway/pie/tree/main/packages/components/pie-cookie-banner/locales)",
-            "Assigns the localisation data for the component strings.",
-            "English language locale"
+            "country",
+            "Any supported country, that is paired with the language [(see these supported locales)](https://github.com/justeattakeaway/pie/tree/main/packages/components/pie-cookie-banner/locales)",
+            "Used (with the language property) to load the localisation data for the component strings. If not provided or unsupported, the default language-country of en-gb will be used.",
+            "GB"
```

Comment:
Please can you make this lowercase, and can you put this and "en" below in code blocks for styling, so it looks like `"gb"`.